app/src/main/java/com/greenfox/tribesoflagopusandroid/fragments/BuildingsFragment.java

```diff
@@ -47,6 +45,8 @@
     FloatingActionMenu buildingsFloatingMenu;
     FloatingActionButton addFarmFloatingButton, addMineFloatingButton, addBarrackFloatingButton;
 
+    View rootView;
```

Comment:
@georgezsiga instead of storing the rootView, store exactly what you use (in this case you only need the adapter)